internal/templating/ingress/templates_ingress.go

```diff
@@ -100,26 +100,16 @@ func GenerateIngressTemplate(
 	}
 	if lValues.EnvironmentType == "production" {
 		// monitoring is only available in production environments
-		additionalAnnotations["monitor.stakater.com/enabled"] = "false"
 		primaryIngress, _ := url.Parse(lValues.Route)
 		// check if monitoring enabled, route isn't autogenerated, and the primary ingress from the .lagoon.yml is this processed routedomain
 		// and enable monitoring on the primary ingress only.
 		if lValues.Monitoring.Enabled && !route.Autogenerated && primaryIngress.Host == route.Domain {
-			additionalLabels["lagoon.sh/primaryIngress"] = "true"
-
-			// only add the monitring annotations if monitoring is enabled
-			additionalAnnotations["monitor.stakater.com/enabled"] = "true"
-			additionalAnnotations["uptimerobot.monitor.stakater.com/alert-contacts"] = "unconfigured"
-			if lValues.Monitoring.AlertContact != "" {
-				additionalAnnotations["uptimerobot.monitor.stakater.com/alert-contacts"] = lValues.Monitoring.AlertContact
-			}
-			if lValues.Monitoring.StatusPageID != "" {
-				additionalAnnotations["uptimerobot.monitor.stakater.com/status-pages"] = lValues.Monitoring.StatusPageID
-			}
-			additionalAnnotations["uptimerobot.monitor.stakater.com/interval"] = "60"
+			additionalLabels["route.lagoon.sh/primary"] = "true"
+		} else {
+			additionalLabels["route.lagoon.sh/primary"] = "false"
 		}
 		if route.MonitoringPath != "" {
-			additionalAnnotations["monitor.stakater.com/overridePath"] = route.MonitoringPath
+			additionalAnnotations["route.lagoon.sh/path-override"] = route.MonitoringPath
```

Comment:
```suggestion
			additionalAnnotations["lagoon.sh/monitoringPath"] = route.MonitoringPath
```
Previously we discussed to use this annotation and this is also used on the ingress operator but if you like to use a different annotation we can certainly change this.